src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -166,7 +167,7 @@ public async Task InitAsync()
                 }
                 catch (TimeoutException)
                 {
-                    if (timeout == InitialConnectionTimeout && _logger.IsInfoEnabled)
```

Comment:
Why `timeout == InitialConnectionTimeout` condition was removed?

Reply:
because then it would only print the log on first register connection try. I want to have an indication if there is subscription worker trying to register connection.

Reply:
ok